tls/s2n_config.h

```diff
@@ -198,6 +203,9 @@ struct s2n_config {
 
     void *renegotiate_request_ctx;
     s2n_renegotiate_request_cb renegotiate_request_cb;
+
+    /* What version of the handshake should be negotiated */
+    s2n_traffic_context traffic_context;
```

Comment:
I'm not sure I understand this comment - how is the context version related to the handshake?